src/Fetch.js

```diff
@@ -490,11 +498,19 @@ function emscripten_start_fetch(fetch, successcb, errorcb, progresscb) {
     else if (errorcb) errorcb(fetch);
   };
 
+  var reportReadyStateChange = function(fetch, xhr, e) {
+#if FETCH_DEBUG
+    console.log('fetch: headers received successfully. e: ' + e);
```

Comment:
The comment looks outdated - i.e. change this comment from `headers received succesfully` to `readyStateChange`?

Reply:
Done.